tests/resources/p2p/test_status.py

```diff
@@ -22,11 +22,28 @@ def setUp(self):
     def test_get(self):
         response = yield self.web.get("status")
         data = response.json_value()
+
         server_data = data.get('server')
         self.assertEqual(server_data['app_version'], 'Hathor v{}'.format(hathor.__version__))
         self.assertEqual(server_data['network'], 'testnet')
         self.assertGreater(server_data['uptime'], 0)
 
+        dag_data = data.get('dag')
+        # We have the genesis block
+        self.assertEqual(len(dag_data['best_block_tips']), 1)
+        self.assertIsNotNone(dag_data['best_block_tips'][0])
+        # As we don't have a type, we most check if the keys are there,
+        # and the types are correct
+        self.assertIn('hash', dag_data['best_block_tips'][0])
+        self.assertIn('height', dag_data['best_block_tips'][0])
+        self.assertIsInstance(dag_data['best_block_tips'][0]['hash'], str)
+        self.assertIsInstance(dag_data['best_block_tips'][0]['height'], int)
+        self.assertIsNotNone(dag_data['best_block'])
+        self.assertIn('hash', dag_data['best_block'])
+        self.assertIn('height', dag_data['best_block'])
+        self.assertIsInstance(dag_data['best_block']['hash'], str)
+        self.assertIsInstance(dag_data['best_block']['height'], int)
```

Comment:
Maybe we should assert if the values returned are correct as well. We are only checking their types and their presence here, but not the values themselves.

Reply:
Mmm... the `height` value is ok to test, it is always `0`. But the `hash` is dynamic and extracting its value would expose the internal process of the endpoint, which is not recommended for this type of test. Maybe testing the length of the `hash` could be an alternative.

Reply:
Since this is the Genesis block, the hash is not dynamic.

The genesis block hash used in the tests is defined in https://github.com/HathorNetwork/hathor-core/blob/master/hathor/conf/unittests.py#L28